src/ranged.cpp

```diff
@@ -304,7 +321,10 @@ int player::fire_gun( const tripoint &target, int shots, item &gun )
         }
 
         dispersion_sources dispersion = get_weapon_dispersion( gun );
-        dispersion.add_range( recoil_total() );
+        // Note 5/8/2019: converting dispersion to a normal distribution amplified the effect of 
```

Comment:
Explanation of a correction like this *is* a good reason to reference the cchange that caused it, though the date is unnecessary, it's there in the commit history.